pkg/client/tx/client.go

```diff
@@ -404,6 +376,36 @@ func (tClient *txClient) goSubscribeToOwnTxs(ctx context.Context) {
 			if len(txErrorChans) == 0 {
 				delete(tClient.txTimeoutPool, timeoutHeight)
 			}
+
+			// Check for a corresponding error channel in the map.
+			// Only close and remove from tClient.txErrorChans if it has been initiated
```

Comment:
Make sure to update the name in the comments as well (if we choose to rename the variable`